libstuff/libstuff.cpp

```diff
@@ -2601,7 +2601,8 @@ string SGetCurrentExceptionName()
     // exception name.
     int status = 0;
     size_t length = 1000;
-    char buffer[length] = {0};
+    char buffer[length];
+    memset(buffer,0,length);
```

Comment:
Let's do this update to follow our style: 
```suggestion
    memset(buffer, 0, length);
```